pyrokinetics/cgyro.py

```diff
@@ -171,6 +171,16 @@ def write(self, pyro, file_name, directory='.'):
         # Numerics
         numerics = pyro.numerics
 
+        if numerics.bpar and not numerics.apar:
+            raise ValueError("Can't have bpar without apar in CGYRO")
+
+        if numerics.bpar:
+            cgyro_input['N_FIELD'] = 3
+        elif numerics.apar:
+            cgyro_input['N_FIELD'] = 2
+        else:
+            cgyro_input['N_FIELD'] = 1
```

Comment:
Not sure if this is more readable, but this could be:

```suggestion
        cgyro_input['N_FIELD'] = 1 + int(numerics.bpar) + int(numerics.apar)
```